newIDE/app/src/LayersList/index.js

```diff
@@ -126,17 +135,40 @@ type State = {|
   effectsEditedLayer: ?gdLayer,
 |};
 
+const hasLightingLayer = (layout: gdLayout) => {
+  const layersCount = layout.getLayersCount();
+  let isLightingLayerPresent = false;
+  mapReverseFor(0, layersCount, i => {
+    const layer = layout.getLayerAt(i);
+    if (layer.isLightingLayer()) isLightingLayerPresent = true;
```

Comment:
nit: `if (xxx) return true`

Reply:
@4ian I'm not so sure about it, `mapReverseFor` basically returns a list, right? I don't really want a list of booleans here, just a confirmation that a lighting layer exists (or not). Am I missing something?

Reply:
Or I could do something like this, 
`return mapReverseFor(0, layersCount, i => {`
`  const layer = layout.getLayerAt(i);`
`  if (layer.isLightingLayer()) return true;`
`  return false;`
` }).reduce((acc, value) => acc || value);`

Maybe it's an overkill, but the intent could be expressed both ways, either this or a variable 🤔

Reply:
ah my bad, of course returning in the closure won't work :) 

Then you can do something like `mapReverseFor(0, layersCount, i => layout.getLayerAt(i).isLightingLayer()).filter(Boolean).length > 0` which is a bit shorter to express the logic of "check if there is already at least one layer that is a lighting layer".